src/AppBundle/Controller/Security/AdherentSecurityController.php

```diff
@@ -46,4 +55,70 @@ public function loginCheck()
     public function logoutAction()
     {
     }
+
+    /**
+     * @Route("/mot-de-passe-oublie", name="adherent_forgot_password")
+     * @Method("GET|POST")
+     */
+    public function forgotPasswordAction(Request $request)
+    {
+        $form = $this->createFormBuilder()
+            ->add('email', EmailType::class, ['constraints' => new NotBlank()])
+            ->add('submit', SubmitType::class)
+            ->getForm()
+        ;
+
+        if ($form->handleRequest($request)->isSubmitted() && $form->isValid()) {
+            $email = $form->get('email')->getData();
+            $adherent = $this->getDoctrine()->getRepository(Adherent::class)->findByEmail($email);
+
+            if ($adherent) {
```

Comment:
`if ($adherent = $this->getDoctrine()->getRepository(Adherent::class)->findByEmail($email)) {`